server/dataset/cxg_dataset.py

```diff
@@ -187,7 +187,7 @@ def _validate_and_initialize(self):
 
     @staticmethod
     def _open_array(uri, tiledb_ctx):
-        with tiledb.Array(uri, mode="r", ctx=tiledb_ctx) as array:
+        with tiledb.open(uri, mode="r", ctx=tiledb_ctx) as array:
```

Comment:
Reading this more closely, can't this entire function be replaced with a single call to tiledb.open?

Reply:
Ha, good point! 😄